recipes/stages/_base_/data/cls_semi_data.py

```diff
@@ -0,0 +1,34 @@
+_base_ = [
```

Comment:
Should it be moved to otx/algo/cls/model/data_pipeline.py?

Reply:
Supervised/incremental is not separated with data_pipeline.py and model.py. I think we need anothet PR to align implementation method for Semi-SL include unlabeled_hook in classification. (also for segmentation)

Reply:
In my opinion, unlabeled_hook is not needed for classification.

Reply:
I suggest to align it by setting up the meeting. Could you summarize the agendas? I will upload PR till the meeting, so we can compare the code change. I personally think that data_pipeline doesn't have to be separated when it globally used across the models and unlabeled_hook is needed.